.github/CODEOWNERS

```diff
@@ -183,12 +180,9 @@
 /cli/jobs/spark/serverless-spark-standalone-user-identity.yaml @sdgilley @msakande @Blackmist @ssalgadodev @lgayhardt @fbsolo-ms1  
 /cli/jobs/spark/spark-job-component.yml @sdgilley @msakande @Blackmist @ssalgadodev @lgayhardt @fbsolo-ms1  
 /cli/misc.sh @sdgilley @msakande @Blackmist @ssalgadodev @lgayhardt @fbsolo-ms1  
-/cli/monitoring/azureml-e2e-model-monitoring @sdgilley @msakande @Blackmist @nemanjarajic @ahughes-msft @yunjie-hub 

```

Comment:
leave these lines as they were.  We don't want to delete it when names are added to ours as was done here.